### DISCUSSION TITLE
Make flag column sortable

### DISCUSSION THREAD

**Discussion by lloc** [lloc/Multisite-Language-Switcher#202] [Ideas]
It would be helpful if the flag-columns in the backend are sortable. In this case we could understand where a translation is missing when there are a lot of elements.

**Comment by orbitaloop** [comment]
+1

**Comment by speccode** [comment]
I guess it needs to separate each language to different column. Maybe better solution will be add filter so you'll be able to choose only posts or pages that are / or not translated?

**Comment by lloc** [comment]
Yes this is probably a better solution.

**Comment by speccode** [comment]
There will be small problem with creating such filter for taxonomies table. For posts I simply used `restrict_manage_posts` action, that is fired in `extra_tablenav` function for `WP_Posts_List_Table`. In taxonomies table class there is no such hook.

I guess we can use `after-{$taxonomy}-table` to add some JavaScript to inject HTML to form, then find way to hookup with taxonomy query. But that sounds more like "hack" not a solution.

**Comment by speccode** [comment]
@lloc so what do you think I should do with this?

**Comment by lloc** [comment]
I would stick to WordPress standard functions. With WordPress 3.8 there are a lot of new things to consider so I would like to wait some days.

**Comment by speccode** [comment]
I'm afraid that I don't see anything new that could help us here. For posts functionality is ready. Do you want pull request only for it?

**Comment by lloc** [comment]
Yes, let's begin with this.
